src/pages/Dashboard/index.jsx

```diff
@@ -1,167 +0,0 @@
-import React from "react";
-import axios from "axios";
-import { Navigate, useNavigate } from "react-router-dom";
-
-import { RiLogoutBoxRLine } from "react-icons/ri";
-import {
-    MdAddCircleOutline,
-    MdOutlineRemoveCircleOutline
-} from "react-icons/md";
-import {AiOutlineClose} from "react-icons/ai";
-
-import {
-    Container,
-    ContainerRegister,
-    OperationContent,
-    ContainerButtons,
-    Button,
-} from "./styles";
-
-import { UserContext } from "../../contexts/UserContext";
-
-export function Dashboard() {
-    const [allFinances, setAllFinances] = React.useState([]);
-    const [total, setTotal] = React.useState();
-    const { token, username } = React.useContext(UserContext);
-
-    const navigate = useNavigate();
-
-    React.useEffect(() => {
-        const promise = axios.get("https://mywallet-api-saulo.herokuapp.com/", {
```

Comment:
Isso aqui é uma redflag grande, você deixou o link do back-end hardcoded, isso é, fixo diretamente no código. Isso é a fonte de uma série de problemas, sendo o principal problema o fato de que toda vez que vc for desenvolver uma nova feature, vc precisa trocar o link do deploy para o link do localhost. Existem várias formas de lidar com isso, mas o princípio é que o link da api deve ser uma variável, pra que possamos alterar facilmente esse caminho.